imgui_internal.h

```diff
@@ -1012,6 +1012,8 @@ struct ImGuiContext
     ImGuiID                 TempInputTextId;                    // Temporary text input when CTRL+clicking on a slider, etc.
     ImGuiColorEditFlags     ColorEditOptions;                   // Store user options for color edit widgets
     ImVec4                  ColorPickerRef;
+    float                   ColorPickerLastHue;
+    float                   ColorPickerLastActiveColor[3];
     bool                    DragCurrentAccumDirty;
```

Comment:
Would rename those two to `ColorEditLastHue` and `ColorEditLastActiveColor[3]` and move above `ColorPickerRef`.